yt_dlp/extractor/funker530.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+
+from .common import InfoExtractor
+from .rumble import RumbleEmbedIE
+from .youtube import YoutubeIE
+from ..utils import clean_html, ExtractorError, strip_or_none, get_element_by_class
```

Comment:
sort class before function, then alphabetically
or use isort
```suggestion
from ..utils import ExtractorError, clean_html, get_element_by_class, strip_or_none
```